tests/golos.vesting_tests.cpp

```diff
@@ -221,6 +221,39 @@ BOOST_FIXTURE_TEST_CASE(buy_vesting, golos_vesting_tester) try {
     CHECK_MATCHING_OBJECT(vest.get_stats(), mvo()("supply", vest.asset_str(buy1+buy2)));
 } FC_LOG_AND_RETHROW()
 
+BOOST_FIXTURE_TEST_CASE(bulk_buy_vesting, golos_vesting_tester) try {
+    BOOST_TEST_MESSAGE("Test bulk buying vesting / converting token to vesting");
+    BOOST_CHECK_EQUAL(success(), token.create(cfg::emission_name, token.make_asset(1000), {cfg::charge_name, cfg::publish_name}));
+    BOOST_CHECK_EQUAL(success(), token.issue(cfg::emission_name, N(sania), token.make_asset(600), "issue tokens sania"));
+    produce_block();
+
+    BOOST_CHECK_EQUAL(success(), vest.create_vesting(cfg::emission_name));
+    BOOST_CHECK_EQUAL(success(), vest.open(N(sania)));
+    BOOST_CHECK_EQUAL(success(), vest.open(N(pasha)));
+    BOOST_CHECK_EQUAL(success(), vest.open(N(tania)));
+    BOOST_CHECK_EQUAL(success(), vest.open(N(vania)));
+    produce_block();
+
+    BOOST_CHECK_EQUAL(success(), token.bulk_transfer( N(sania), {{cfg::vesting_name, token.make_asset(200), golos::config::send_prefix + name{"pasha"}.to_string()},
+                                                                 {cfg::vesting_name, token.make_asset(100), golos::config::send_prefix + name{"vania"}.to_string()}}
+                                                    ));
+
+    BOOST_CHECK_EQUAL(success(), token.bulk_transfer( N(sania), {{cfg::vesting_name, token.make_asset(50), "buy vesting"},
```

Comment:
It's too easy example. To check token/vesting convertion accuracy we need:
* big token amount in transfer
* several transfers in bulk
* check **total vesting equal sum of vesting on all accounts** after transfer